scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1827,9 +1789,10 @@ if [[ -z $MYSQL_VERSION ]]; then
 fi
 
 # Verify PXB versions we have
-verify_pxb_version "${XTRABACKUP_THIS_VER_PATH}" "${XB_THIS_REQUIRED_VERSION}"
-verify_pxb_version "${XTRABACKUP_PREV_VER_PATH}" "${XB_PREV_REQUIRED_VERSION}"
-verify_pxb_version "${XTRABACKUP_PREV_LTS_VER_PATH}" "${XB_PREV_LTS_REQUIRED_VERSION}"
+# KH: temporary
+#verify_pxb_version "${XTRABACKUP_THIS_VER_PATH}" "${XB_THIS_REQUIRED_VERSION}"
+#verify_pxb_version "${XTRABACKUP_PREV_VER_PATH}" "${XB_PREV_REQUIRED_VERSION}"
+#verify_pxb_version "${XTRABACKUP_PREV_LTS_VER_PATH}" "${XB_PREV_LTS_REQUIRED_VERSION}"
```

Comment:
PXB-8.4 is already released. Does still comment still hold good?

Reply:
done. Leftover from the time when pxb 8.4 was not ready yet